tests/phpt/migration_php8/006_is_numeric.php

```diff
@@ -0,0 +1,13 @@
+@kphp_runtime_should_warn
+
```

Comment:
It's better to specify warning message here

Reply:
Yep, done.